vendor/manifest

```diff
@@ -1460,10 +1460,10 @@
 		},
 		{
 			"importpath": "github.com/weaveworks/tcptracer-bpf",
-			"repository": "https://github.com/weaveworks/tcptracer-bpf",
+			"repository": "https://github.com/kinvolk/tcptracer-bpf",
 			"vcs": "git",
-			"revision": "b715a3b635b8d9c4a096bbd6009826b57fe64c38",
-			"branch": "master",
+			"revision": "3b09ef1351d865c0b2519e1486f6c88db3905780",
+			"branch": "alban/fdinstall",
```

Comment:
The vendoring was generated by updating the "repository", "revision" and "branch" fields and running the command:
```
gvt restore --filter=github.com/weaveworks/tcptracer-bpf
```
using `gvt` patched with https://github.com/FiloSottile/gvt/pull/93